examples/gno.land/p/demo/grc/grc20/account_banker.gno

```diff
@@ -0,0 +1,70 @@
+package grc20
+
+import (
+	"std"
+)
+
+func PrevRealmBanker(b *Bank) *fnBanker {
+	return &fnBanker{
+		accountFn: func() std.Address {
+			caller := std.PrevRealm().Addr()
+			return caller
+		},
+		bank: b,
+	}
+}
+
+func ReadonlyBanker(b *Bank) *fnBanker {
```

Comment:
As I mentioned, I think the ReadonlyBanker could simply be the `Bank` / `Token` itself.

Reply:
The ReadonlyBanker is similar to the Readonly Banker for an std.Coin. It is not about being "safe" or not; rather, it involves having (and being able to share) a pointer that can only be used for read-only operations.

We can have readonly safe objects and read-write safe objects.